src/test/java/com/example/demo/controller/GreetingControllerTests.java

```diff
@@ -1,19 +1,23 @@
 package com.example.demo.controller;
 
+import com.example.demo.services.GreetingService;
 import org.junit.jupiter.api.Test;
-import org.mockito.Mockito;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-
-import com.example.demo.services.GreetingService;
+import static org.mockito.Mockito.when;
 
 class GreetingControllerTests {
+	@Mock
+	private GreetingService greetingService;
+	@InjectMocks
+	private GreetingController greetingController;
 
 	@Test
 	void greetingEndpoint() {
```

Comment:
Please change method name to more descriptive one, e.g. "testGreetingEndpoint" or "shouldReturnGreetingWithName"